pyproject.toml

```diff
@@ -102,3 +102,15 @@ doctest_optionflags = [
     "NORMALIZE_WHITESPACE",
     "IGNORE_EXCEPTION_DETAIL",
 ]
+
+[tool.mypy]
+ignore_errors = true
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+module = [
```

Comment:
Here, we can keep adding the modules we want to check. Eventually, we would like to simply remove this and check everything